src/main/java/reposense/wizard/prompt/UntilPrompt.java

```diff
@@ -0,0 +1,25 @@
+package reposense.wizard.prompt;
+
+import reposense.wizard.InputBuilder;
+
+/**
+ * Represents a Prompt to get the untilDate flag.
+ */
+public class UntilPrompt extends Prompt {
+    private static final String DESCRIPTION = "Enter the end date for the period to be analyzed";
+    private static final String FORMAT = "DD/MM/YYYY";
+
+    public UntilPrompt() {
+        super(DESCRIPTION, FORMAT);
+    }
+
+    @Override
+    public InputBuilder addToInput(InputBuilder inputBuilder) {
+        return inputBuilder.addUntilDate(getResponse());
```

Comment:
With regard to error handling, since the DES implementation stores the current configuration in a `StringBuilder`, is it impossible to check for errors w.r.t earlier inputs? (e.g. until date earlier than start date)

I personally don't think this is super important but was just wondering if something like this is possible for the current design.

Reply:
That is an interesting point.

I suppose one possible solution would be to group the Since and Until prompt into a single prompt that stores the state for both sinceDate and untilDate, and as such we can check for correctness from there on.

It may also be possible for us to define the dates as attributes/states within the actual WizardRunner, though I believe this can make things very messy as we will have a lot of attributes within the WizardRunner itself, and is less extensible to different possible versions of Wizards such as possibly an AdvancedWizard in the future.